packages/core/admin/admin/src/translations/en.json

```diff
@@ -678,6 +678,7 @@
   "content-manager.components.SettingsViewWrapper.pluginHeader.description.edit-settings": "Customize how the edit view will look like.",
   "content-manager.components.SettingsViewWrapper.pluginHeader.description.list-settings": "Define the settings of the list view.",
   "content-manager.components.SettingsViewWrapper.pluginHeader.title": "Configure the view — {name}",
+  "content-manager.components.SelectedEntriesModal.published": "Already Published",
```

Comment:
I think the `components` doesn't make too much sense here, because it refers to content editing components. What do you think about `content-manager.bulk-publish.already-published` instead?